interface/patient_file/summary/stats_full.php

```diff
@@ -131,191 +123,203 @@ function newEncounter() {
+                                        echo "<tr><td colspan=\"" . $totalCol . "\"><b>" . xlt("None") . "</b></td></tr>";
+                                    } else {
+                                        // Data entry has not happened to this type, so can show the none selection option.
+                                        echo "<tr><td colspan=\"{$totalCol}\"><input type='checkbox' class='noneCheck' name='" .
+                                        attr($focustype) . "' value='none'";
+                                        if (!acl_check_issue($focustype, '', 'write')) {
+                                            echo " disabled";
+                                        }
 
-                    // calculate the status
-                    if ($row['outcome'] == "1" && $row['enddate'] != null) {
-                          // Resolved
-                          $statusCompute = generate_display_field(array('data_type'=>'1','list_id'=>'outcome'), $row['outcome']);
-                    } elseif ($row['enddate'] == null) {
-                          $statusCompute = xlt("Active");
-                    } else {
-                          $statusCompute = xlt("Inactive");
-                    }
-
-                    $click_class='statrow';
-                    if ($row['erx_source']==1 && $focustype=='allergy') {
-                        $click_class='';
-                    } elseif ($row['erx_uploaded']==1 && $focustype=='medication') {
-                        $click_class='';
-                    }
-
-                    echo " <tr class='" . attr($bgclass) . " detail' $colorstyle>\n";
-                    echo "  <td style='text-align:left' class='" . attr($click_class) . "' id='" . attr($rowid) . "'>" . text($disptitle) . "</td>\n";
-                    echo "  <td>" . text(oeFormatShortDate($row['begdate'])) . "&nbsp;</td>\n";
-                    echo "  <td>" . text(oeFormatShortDate($row['enddate'])) . "&nbsp;</td>\n";
-                    // both codetext and statusCompute have already been escaped above with htmlspecialchars)
-                    echo "  <td>" . $codetext . "</td>\n";
-                    echo "  <td>" . $statusCompute . "&nbsp;</td>\n";
-                    echo "  <td class='nowrap'>";
-                    echo generate_display_field(array('data_type'=>'1','list_id'=>'occurrence'), $row['occurrence']);
-                    echo "</td>\n";
-                    if ($focustype == "allergy") {
-                          echo "  <td>";
-                            echo generate_display_field(array('data_type'=>'1','list_id'=>'reaction'), $row['reaction']);
-                          echo "</td>\n";
+                          echo " /><b>" . xlt("None{{Issue}}") . "</b></td></tr>";
                     }
-
-                    echo "  <td>" . text($row['referredby']) . "</td>\n";
-                    echo "  <td>" . text(oeFormatDateTime($row['modifydate'])) . "</td>\n";
```

Comment:
Why need to remove the locale setting date formatting?

Reply:
Whoops, since I submitted the PR that file developed merge conflicts, I resolved them but missed this. Pushing a fix now.